tests/suites/test_suite_platform.function

```diff
@@ -84,7 +84,13 @@ void time_delay_seconds(int delay_secs)
     sleep_ms(delay_secs * 1000);
 
     elapsed_secs = mbedtls_time(NULL) - current;
-    TEST_ASSERT(elapsed_secs >= delay_secs && elapsed_secs < 4 + delay_secs);
+
+    /* Built-in mbedtls_time function returns the number of seconds since the
+     * Epoch. That is affected by discontinuous jumps and cause test fail.
```

Comment:
```suggestion
     * Epoch. That is affected by discontinuous jumps and in case system wake up earlier, the test case will fail.
```
Seems if the system wakes up early, the elapsed_secs will be smaller than delay_secs.   Can we change the description like above?

Reply:
This description is wrong. That is due to different clock source. 

Add more description to reduce the misunderstand on it.